src/scripts/main.js

```diff
@@ -1 +1,69 @@
 'use strict';
+
+function handleNotification(divMessage, isSuccess) {
+  const div = document.createElement('div');
+
+  div.textContent = divMessage;
+  div.className = isSuccess ? 'success' : 'error';
+}
+
+const firstPromise = new Promise((resolve, reject) => {
+  document.addEventListener('click', () => {
+    resolve('First promise was resolved');
+  });
+
+  setTimeout(() => {
+    reject(new Error('First promise was rejected'));
+  }, 3000);
+});
+
+const secondPromise = new Promise((resolve, reject) => {
+  document.addEventListener('mousedown', (e) => {
+    if (e.button === 0 || e.button === 2) {
+      resolve('Second promise was resolved');
+    }
+  });
+});
+
+const thirdPromise = new Promise((resolve, reject) => {
+  let leftClicked = false;
+  let rightClicked = false;
+
+  document.addEventListener('mousedown', (e) => {
+    if (e.button === 0) {
+      leftClicked = true;
+    }
+
+    if (e.button === 2) {
+      rightClicked = true;
+    }
+
+    if (leftClicked && rightClicked) {
+      resolve('Third promise was resolved');
+    }
+  });
+});
```

Comment:
The third promise logic for resolving when both left and right clicks are detected may not work as intended. The event listeners for 'mousedown' do not reset the `leftClicked` and `rightClicked` flags, so once both are set to true, the promise will resolve on any subsequent click. Consider resetting these flags appropriately.